README.md

```diff
@@ -105,22 +108,37 @@ client.ExportEnv(client.ResolveEnv());
 ```
 
 ### Exceptions
-Any error encountered by the SecretHub client will be thrown as an `ApplicationException`. The full error message can be retrieved from the `Message` field.
+Any error encountered by the SecretHub client will be thrown as an `Exception`. The full error message can be retrieved from the `Message` field.
 ```csharp
 try 
 {
 	string secret = client.Read("path/to/secret");
 } 
-catch(ApplicationException ex)
+catch(Exception ex)
 {
 	Console.WriteLine(ex.Message);
 }
 ```
 
+### Credential
+To use the SecretHub .NET client, you need to provide a credential for your __SecretHub__ account.
+You can sign up for a free developer account [here](https://signup.secrethub.io/).
+
+After signup, the credential is located at `$HOME/.secrethub/credential` by default.
```

Comment:
I'm not sure if this is the right place to comment this, but maybe we should also say: or `C:\Users\<username>\.secrethub\credential` on Windows` as I imagine a lot of the users of the .NET client work on/with Windows.

Reply:
Feel free to suggest an update :+1: